task/git-version/0.1/tests/run.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: git-version-test
+spec:
+  workspaces:
+    - name: shared-workspace
+  tasks:
+    - name: git-fetch-all
+      taskRef:
+        name: git-cli
```

Comment:
cant we use git-clone task here

Reply:
No, git-clone doesn't "fetch all" branches which is needed for the git version tool.